scanners/nessus/nessus_none.py

```diff
@@ -93,6 +102,25 @@ def scan_id(self) -> int:
             raise RuntimeError("scan_id is None")
         return self._scan_id
 
+    @generic_authentication_factory()
+    def authentication_factory(self):
+        """This is the default function, attached to error reporting"""
+        raise RuntimeError(f"No valid authenticator found for {self}")
```

Comment:
The error might be friendlier for users if it just printed what config option is not supported for the given scanner, so they have actionable info. So like "authentication.type: 'oauth2_rtoken' not supported with nessus scanner", dumping self is a bit noisy